Star component

DimaMklk opened the pull request.

Commit:
Star component

barbarosh commented on client/components/Star/Star.js:
```diff
@@ -0,0 +1,43 @@
+
+function Star(props) {
+  let fill, stroke = '';
+  if (props.state === 'checked') {
```
@DimaMklk Должно быть props.checked равно true или false

barbarosh replied:
А лучше так 

```
let fill='none', stroke = '#434448';
if (props.checked) {
fill = 'url(#lgrad)';
stroke = 'none';
}
`
```

barbarosh replied:
@DimaMklk

DimaMklk replied:
@barbarosh Done

Commit:
Star component *modificated

Also changed chart's multiplier.
I don't know why, but `checked="false"` and `if (props.checked)` gives
me `true`. So I wrote `if (props.checked === true)`
